### PR TITLE
Fix for "No tables found" problem on deleting tables in the last tab, #14205

### LINKED ISSUE #14205: "No tables found in database" when you delete all tables from last page
### Steps to reproduce
1. Create enough tables, so you have 2 pages
2. Delete all tables from 2nd page
3. "You get message: No tables found in database"

### Expected behaviour
You should see tables from 1st page

### PR DESCRIPTION
**Pull request by aroralakshya**
Fix #14205

Signed-Off-By: Lakshya arora <arora.lakshya123@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

**Comment by ibennetch** [comment]
I apparently have better luck than Maurício did; it worked for me. After deleting the table, the navigation pane did not update; I had to manually use the `<` button to go to the previous page, but I did not get an error.

I wonder if a better solution would be to automatically refresh/move the user to the new last page.

This isn't a bad or wrong solution as-is, but I wonder if we can do better for presenting a smooth interface to the user.

**Comment by nulll-pointer** [comment]
@mauriciofauth  @ibennetch , try hard reload. Probably, it's loading the script from the cache.

**Review by MauricioFauth — COMMENTED** [review]
I tried again, checked that the script had been loaded correctly, but the issue keeps happening.

**Comment by nulll-pointer** [comment]
I'm looking into it.

**Comment by ibennetch** [comment]
I know this one wasn't part of the GSoC plan, but did you figure out what was wrong?

File: js/navigation.js [hunk 1]
```diff
@@ -259,6 +259,9 @@ function traverseNavigationForPaths () {
                     .find('span.pos2_value:last')
                     .text();
             }
+            if( $('#pma_navigation_tree').find('li.database.selected').children('div.list_container').find('li.table').length === 1) {
```
Comment (mauriciofauth) [inline js/navigation.js]: ESLint failed. See: https://travis-ci.org/phpmyadmin/phpmyadmin/jobs/414006024#L744

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

File: js/navigation.js (hunk 1 above)
Comment (nulll-pointer) [reply js/navigation.js]: I've fixed it, kindly have a look.

**Comment by nulll-pointer** [comment]
@mauriciofauth, fixed

**Review by MauricioFauth — COMMENTED** [review]
@nulll-pointer Now I realize that we are not seeing the same thing.
You are looking at the list of tables in the navigation menu. However, the issue is about the list of tables in the database Structure tab.
You can see the pagination if you change the maximum number of tables, for example: `$cfg['MaxTableList'] = 2;`
Now go to the last page and drop all the tables. You will see the message _No tables found in database._
The only ways I found to stop seeing this message was by changing the database or manually adding the `pos` parameter, for example: `db_structure.php?db=sakila&pos=0`

**Comment by nulll-pointer** [comment]
oh, yes. But, this was also a bug :smile:

**Comment by mauriciofauth** [comment]
@nulll-pointer Yes, you've found a related bug. :)

**Comment by williamdes** [comment]
@sudo-bot Deploy PR

**Review by williamdes — CHANGES_REQUESTED** [review]
This PR did not solve the issue from my tests

**Comment by williamdes** [comment]
Closing since this PR does not solve the issue, sorry.

**Pull request closed without merging**